### PR TITLE
Modified the user serializer as requested.

### PR DESCRIPTION
**Pull request by pythondjay**
Modified the user serializer to overwrite the create method as the password was not being hashed in the database.

### CODE REVIEW

File: apps/user/serializers.py [hunk 1]
```diff
@@ -4,6 +4,22 @@
...
+        )
+
+        return user
+
     class Meta:
         model = models.User
-        fields = "__all__"
+        fields = (
+            "username",
+            "password",
```
Comment (Termuellinator) [inline apps/user/serializers.py:23]: Won't that show the hashed password in the response body?

**Review by Termuellinator — COMMENTED** [review]
Looks good besides the comment for the password. Will try to test later.

File: apps/user/serializers.py (hunk 1 above)
Comment (pythondjay) [reply apps/user/serializers.py:23]: No, it didn't show the password when I was testing it. It only showed the id, username, email and date joined. When I started the server it wouldn't work until I had password in fields.

Comment (Termuellinator) [reply apps/user/serializers.py:23]: Oh, right...sorry, probably because of the `write_only=True` i think.

**Review by Termuellinator — APPROVED** [review]
Seems to work fine here, too. Good job!

**Pull request merged**